src/plugins/lock/README.md

```diff
@@ -0,0 +1,15 @@
+- infos = Information about the lock plugin is in keys below
+- infos/author = Kurt Micheli <e1026558@student.tuwien.ac.at>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides =
+- infos/placements = pregetstorage postcommit
+- infos/status = maintained unittest tested nodep libc 0
+- infos/description =
+
+## LOCK ##
+
+This small plugin lock for each file. The granularity is huge and the logic very simple.
```

Comment:
Yes, looks like. But it was intended to be *only* a global plugin.

The idea is to completely lock down any access do the database if a single `kdbSet` is active. Use some global file, e.g. /run/elektra/kdb.pid